target/linux/kirkwood/patches-5.10/108-dns325.patch

```diff
@@ -0,0 +1,116 @@
+--- a/arch/arm/boot/dts/kirkwood-dnskw.dtsi
++++ b/arch/arm/boot/dts/kirkwood-dnskw.dtsi
+@@ -31,7 +31,7 @@
+ 		};
+ 	};
+ 
+-	gpio_fan {
++	fan0: gpio_fan {
+ 		/* Fan: ADDA AD045HB-G73 40mm 6000rpm@5v */
+ 		compatible = "gpio-fan";
+ 		pinctrl-0 = <&pmx_fan_high_speed &pmx_fan_low_speed>;
+@@ -189,33 +189,18 @@
+ 
+ 	partition@0 {
+ 		label = "u-boot";
+-		reg = <0x0000000 0x100000>;
++		reg = <0x0000000 0x0e0000>;
+ 		read-only;
+ 	};
+ 
+-	partition@100000 {
+-		label = "uImage";
+-		reg = <0x0100000 0x500000>;
+-	};
+-
+-	partition@600000 {
+-		label = "ramdisk";
+-		reg = <0x0600000 0x500000>;
+-	};
+-
+-	partition@b00000 {
+-		label = "image";
+-		reg = <0x0b00000 0x6600000>;
+-	};
+-
+-	partition@7100000 {
+-		label = "mini firmware";
+-		reg = <0x7100000 0xa00000>;
++	partition@e0000 {
++		label = "u-boot environment";
```

Comment:
Maybe `u-boot-env`? Many devices uses that name.

Reply:
done